README.md

```diff
@@ -81,52 +90,47 @@ We have made AWS credentials available in the following file, with the appropria
+3. *Classification challenge*: Classify a hex as sparsly or densely populated, solely based on the service request data. Provide an explantion as to how you're using the data to perform this classification.
+4. *Anomaly Detection challenge*: Reshape the data into the number of requests created per department, per day. Please identify any days in the past 6 months where an anamolous number of requests were created for a particular department. Please describe how you would motivate to the director of that department why they should investigate that anamoly. Your argument should rely upon the contents of the dataset and/or your anamoly detection model.
 
 Feel free to use any other data you can find in the public domain, except for task (3).
 
 **The final output of the execution of your code should be a self-contained `html` file or executed `ipynb` file that is your report.** 
  
 A statistically minded layperson should be able to read this report and follow your analysis without guidance.
 
-Please log the time taken to perform the operations described, and within reason, try to optimise latency and computation resources used.
+Please log the time taken to perform the operations described, and within reason, try to optimise latency and computation resources used. Please also note the comments above with respect to the nature of work that we expect from data scientists.
 
 ### 5. Further Data Transformations (if applying for a Data Engineering Position)
-Write a script which anonymises the `sr_hex.csv` file, but preserves the following resolutions (You may use H3 indexes or lat/lon coordinates for your spatial data):
+Write a script which anonymises the `sr_hex.csv` file, but preserves the following precisions (You may use H3 indexes or lat/lon coordinates for your spatial data):
    * location accuracy to within approximately 500m 
    * temporal accuracy to within 6 hours
-   * scrubs any columns which may contain personally identifiable information.
-
-We expect in the accompanying report that follows you will justify as to why this data is now anonymised. Please limit this commentary to less than 500 words. If your code is written in a code notebook such as Jupyter notebook or Rmarkdown, you can include this commentary in your notebook.
-
-### 6. Data Loading Tasks (if applying for a Data Engineering Position)
-Select a subset of columns (including the H3 index column) from the `sr_hex.csv` or the anonymised file created in the task above, and write it to the write-only S3 bucket. 
+ 
+Please also remove any columns which you believe could lead to the resident who made the request being identified. We expect in the accompanying report that you will justify as to why this data is now anonymised. Please limit this commentary to less than 500 words. If your code is written in a code notebook such as Jupyter notebook or Rmarkdown, you can include this commentary in your notebook.
```

Comment:
I would like a data engineering question where we give them some funny attributes tables and ask them to do a more complex join, possibly involving a groupby / window function or time bound or spatial join